packages/react-integration/demo-app-ts/src/components/demos/NotificationDrawerDemo/NotificationDrawerBasicDemo.tsx

```diff
@@ -29,41 +32,50 @@ export class BasicNotificationDrawerDemo extends React.Component<
   constructor(props: NotificationDrawerProps) {
     super(props);
     this.state = {
-      isOpen: false,
-      isActive: ''
+      isOpen0: false,
+      isOpen1: false,
+      isOpen2: false,
+      isOpen3: false,
+      isOpen4: false
     };
   }
-
-  onToggle = (isOpen: boolean) => {
+  onToggle0 = (isOpen0: boolean) => {
     this.setState({
-      isOpen
+      isOpen0
     });
   };
-
-  onSelect = (event: any) => {
+  onToggle1 = (isOpen1: boolean) => {
     this.setState({
-      isOpen: !this.state.isOpen,
-      isActive: ''
+      isOpen1
     });
-    this.onFocus(event.target.id);
   };
-
-  onClick = (event: any) => {
+  onToggle2 = (isOpen2: boolean) => {
     this.setState({
-      isActive: event.target.id
+      isOpen2
     });
-    this.onFocus(event.target.id);
   };
-
-  onFocus = (id: string) => {
-    if (id) {
-      const element = document.getElementById(id);
-      element.focus();
-    }
+  onToggle3 = (isOpen3: boolean) => {
+    this.setState({
+      isOpen3
+    });
+  };
+  onToggle4 = (isOpen4: boolean) => {
+    this.setState({
+      isOpen4
+    });
+  };
+  onSelect = (event: any) => {
```

Comment:
Can we type this? Is it `React.SyntheticEvent<HTMLDivElement>`?

Reply:
Fixed. I am not sure whether it is HTMLDivElement so I use `React.SyntheticEvent<HTMLElement>` instead.

Reply:
Cool! Yea whatever makes most sense. Ty!